pkgs/development/tools/go-task/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, buildGoPackage }:
+
+let
+  owner =
+    "go-task";
+
+  repo =
+    "task";
+
+in
+buildGoPackage rec {
+  pname   = "go-task";
+  version = "2.8.1";
+
+  src = fetchFromGitHub {
+    inherit owner repo;
+
+    rev    = "v${version}";
+    sha256 = "1yf5s22q4hsxvgzrj4k6gbbq0whl5b9j12mbwimxaf53q535wy4q";
+  };
+
+  goPackagePath = "github.com/${owner}/${repo}";
+
+  goDeps = ./deps.nix;
+
+  postPatch = ''
+    substituteInPlace cmd/task/task.go \
+      --replace 'version = "master"' 'version = "${version}"'
+  '';
+
+  meta = with stdenv.lib; {
+    homepage    = "https://taskfile.dev/";
+    description = "A task runner / simpler Make alternative written in Go";
+    license     = licenses.mit;
+    platforms   = [ "x86_64-linux" ];
```

```suggestion
    platforms   = platforms.all;
```

Most Go programs are multiplatform. Any reason to restrict to x86_64 Linux? Try `platforms.all` and see if see builds it on AArch64 and Darwin.

---

Good catch, I had that when it was originally pulling the linux tarball, fixed